### PR TITLE
chore: cache cni v1.1.8

### PR DESCRIPTION
**Pull request by pjohnst5**
Bake v1.1.8 cni for windows and linux

### CODE REVIEW

**Comment by pjohnst5** [comment]
@xuto2 could you please approve?

File: vhdbuilder/packer/configure-windows-vhd.ps1 [hunk 1]
```diff
@@ -150,9 +150,9 @@ function Get-FilesToCacheOnVHD {
             "https://acs-mirror.azureedge.net/kubernetes/v1.19.2/windowszip/v1.19.2-1int.zip"
         );
         "c:\akse-cache\win-vnet-cni\" = @(
-            "https://acs-mirror.azureedge.net/azure-cni/v1.1.2/binaries/azure-vnet-cni-singletenancy-windows-amd64-v1.1.2.zip",
```
Comment (xuto2) [inline vhdbuilder/packer/configure-windows-vhd.ps1:153]: @AbelHu  FYI  I will merge this PR for now, but please double check if it's ok to delete this or not.

Comment (paulgmiller) [reply vhdbuilder/packer/configure-windows-vhd.ps1:153]: Yeah wiping this is fine.

Comment (AbelHu) [reply vhdbuilder/packer/configure-windows-vhd.ps1:153]: @xuto2 @paulgmiller Deleting an old cache does not impact functionality but may impact the perf for some old clusters using CNI v1.1.2.
I planed to get some data via logs later to decide when we can remove some cached packages.

Comment (AbelHu) [reply vhdbuilder/packer/configure-windows-vhd.ps1:153]: Since new clusters are using toggle for CNI version, it should not impact the perf.

**Pull request merged**